lib/aemmultipart.js

```diff
@@ -63,10 +63,12 @@ async function uploadAEMMultipartFile(filepath, target, options) {
         throw Error(`File '${filepath}' is too large to upload: ${size} bytes, maxPartSize: ${target.maxPartSize} bytes, numUploadURIs: ${target.urls.length}`);
     }
 
+    // Default to the maxPartSize
     // Limit the lower bound to the minPartSize, since the estimated file size to generate uris
     // may be significantly larger than the actual file size. Also make sure that partSize is positive.
     partSize = Math.max(
         partSize,
+        target.maxPartSize || 1,
         target.minPartSize || 1,
         1
     );
```

Comment:
This doesn't really default it to maxPartSize, it will just pick the maxPartSize regardless of everything else. This is temporary right?  If not, you could reduce it down to:

```
partSize = target.maxPartSize || partSize
```

And remove the `partSize` option.

Reply:
I'm not against the change, btw, just that if that's the intent then just remove the rest of the complexity in here so mere mortals can understand it :)